src/Sylius/Behat/Page/Shop/Account/LoginPage.php

```diff
@@ -48,7 +48,7 @@ protected function getDefinedElements(): array
             'login_button' => '[data-test-button="login-button"]',
             'password' => '[data-test-login-password]',
             'username' => '[data-test-login-username]',
-            'validation_error' => '[data-test-flash-message="negative"]',
+            'validation_error' => '[data-test-sylius-flash-message]',
```

Comment:
```suggestion
            'flash_message' => '[data-test-sylius-flash-message]',
```
For the sake of consistency.